client/pages/sections/granted/summary-table.js

```diff
@@ -1,91 +1,168 @@
+                  <td className={classnames({ 'is-last-row': isLastRow })}>{ s['maximum-animals'] }</td>
+                  <td className={classnames({ 'is-last-row': isLastRow })}>{ s['maximum-times-used'] }</td>
                 </Fragment>
               )
           }
-          <th>Life stages</th>
-          <th>GA?</th>
-          <th>Severity category</th>
-        </tr>
-      </thead>
-      <tbody>
-        {
-          (protocols || []).map((protocol, index) => {
-            const species = !isLegacy
-              ? filterSpeciesByActive(protocol, project)
-              : (protocol.species || []).map(s => {
-                const label = getLegacySpeciesLabel(s);
-                return {
-                  ...s,
-                  name: label || '-'
-                };
-              });
-            return species.map((s, speciesIndex) => (
-              <tr key={index + speciesIndex}>
-                {
-                  (speciesIndex === 0) && (
-                    <Fragment>
-                      <td rowSpan={species.length || 1}>{ index + 1 }</td>
-                      <td rowSpan={species.length || 1}>{ protocol.title }</td>
-                    </Fragment>
-                  )
-                }
-                <td>{ s.name }</td>
-                {
-                  isLegacy
-                    ? <td>{s.quantity}</td>
-                    : (
-                      <Fragment>
-                        <td>{ s['maximum-animals'] }</td>
-                        <td>{ s['maximum-times-used'] }</td>
-                      </Fragment>
-                    )
-                }
-                <td>
-                  {
-                    isLegacy
-                      ? s['life-stages']
-                      : (s['life-stages'] || []).join(', ')
-                    }
-                </td>
-                {
-                  isLegacy && <td>{s['genetically-altered'] === true ? 'Yes' : 'No'}</td>
-                }
+          <td className={classnames({ 'is-last-row': isLastRow })}>
+            {
+              isLegacy
+                ? s['life-stages']
+                : (s['life-stages'] || []).map(titleCase).join(', ')
+              }
+          </td>
+          {
+            isLegacy && <td>{s['genetically-altered'] === true ? 'Yes' : 'No'}</td>
+          }
+          {
+            speciesIndex === 0 && (
+              <Fragment>
                 {
-                  speciesIndex === 0 && (
-                    <Fragment>
-                      {
-                        !isLegacy && <td rowSpan={species.length || 1}>{ protocol.gaas === true ? 'Yes' : 'No' }</td>
-                      }
-                      <td rowSpan={species.length || 1}>{ protocol.severity }</td>
-                    </Fragment>
-                  )
+                  !isLegacy && <td rowSpan={species.length || 1} className="is-last-row">{ protocol.gaas === true ? 'Yes' : 'No' }</td>
```

Comment:
Given that the majority of the cells in the row are either wrapped in - or contain - an `isLegacy ? ... ` would it be cleaner to do it once, and have a fragment for each option?

i.e.

```
isLegacy ?
  <Fragment>
    <td>...</td>
    <td>...</td>
    <td>...</td>
  </Fragment> :
  <Fragment>
    <td>...</td>
    <td>...</td>
    <td>...</td>
  </Fragment>
```

Reply:
probably - none of the isLegacy stuff is new and I didn't want to wander too far out of my lane

Reply:
I'll do the thing.

Reply:
TBF half of them are dependent on legacy, and half not, so refactoring would result in half the rows being repeated. I'm not sure that's preferable

Reply:
I'm fairly sure I prefer it, but it's hard to say without seeing it.

Reply:
It looks like it's the middle 4 cells in the row are all dependent on legacy-ness, so only include those in the condition, and leave the first two and last cells out and there shouldn't be much repetition.